.gitignore

```diff
@@ -1 +1,4 @@
 drone-openapi
+coverprofile.out
+testdata/specs/hello_world_spec.json
+testdata/specs/users_spec.json
```

Comment:
Do you want the json files?

Reply:
No, the json actually gets generated during the test from the ymls which are checked in